modules/instruments/php/instrumentqueryengine.class.inc

```diff
@@ -371,6 +371,7 @@ class InstrumentQueryEngine implements \LORIS\Data\Query\QueryEngine
                         $candData[$dict->getName()] = [];
                     }
                     $sid = $loadedInstrument->getSessionID();
+<<<<<<< HEAD
```

Comment:
More unresolved conflicts

Reply:
I was expecting you to resolve that one uless you want me to break it